main/src/ca/uwaterloo/flix/language/phase/unification/JvmUnification.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 Matthew Lutze
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ca.uwaterloo.flix.language.phase.unification
+
+import ca.uwaterloo.flix.language.ast.shared.Scope
+import ca.uwaterloo.flix.language.ast.{RigidityEnv, Type}
+
+object JvmUnification {
```

Comment:
Not sure what this class is about?

Reply:
This is to fulfill the principle that every kind gets its own XyzUnification. Obviously not a lot happens here, but I think it's nice to delegate anyway.

Reply:
OK